locales/en.js

```diff
@@ -44,7 +44,9 @@ const en = {
     confirmMessageTitle: 'Confirm Action',
     confirmMessage: 'Are you sure?',
     yes: 'Yes',
-    no: 'No'
+    no: 'No',
+    manageVisual: 'Manage Visual',
```

Comment:
Maybe move this localization text to the ManageVisualScreen as it is not really shared between pages.